scss/modules/_nav.scss

```diff
@@ -214,18 +219,32 @@
       width: 100%;
     }
 
+    .site-nav__subtitle {
+      border-top: 1px solid $primary-contrast;
+      padding: 2rem 2rem 0 2rem;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `2rem 2rem 0` instead of `2rem 2rem 0 2rem`